### ISSUE TITLE
NTR: Cold pressor test with subnormal blood pressure increase (Autonomic)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7779]
New term request Cold pressor test with subnormal blood pressure increase (elthat_130220195449): Autonomic function; ice

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000323
- label: Reduced sympathoneural responses to the cold pressor test
- parents: [Abnormal autonomic nervous system physiology (HP:0012332)](https://hpo.jax.org/app/browse/term/HP:0012332)
- pmid: PMID:33712946
- def: Response to the cold pressor test below the lower limit of normal. The cold pressor test is a simple and validated test in which the subject immerses one hand or foot into ice water for 1 to 3 min utes while blood pressure (BP) and heart rate are monitored. The cold stimulus activates afferent sensory pathways that, in turn, trigger a sympathetic response resulting in an increase in BP.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7779
- synonyms:
